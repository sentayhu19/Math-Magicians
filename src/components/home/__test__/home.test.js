import React from 'react';
import { render } from '@testing-library/react';
import renderer from 'react-test-renderer';
import Home from '../home';

describe('Home page of the App', () => {
  test('Builds the snapchot of the Home component safely', () => {
    const rend = renderer.create(<Home />);
    expect(rend.toJSON()).toMatchSnapshot();
  });
  test('Render Home component', () => {
    render(<Home />);
  });
});
