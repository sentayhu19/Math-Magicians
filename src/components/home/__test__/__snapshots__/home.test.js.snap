// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page of the App Builds the snapchot of the Home component safely 1`] = `
Array [
  <h2>
    Welcome To Our Page!
  </h2>,
  <p>
    As time passed, many mathematicians gravitated towards universities . An emphasis on free thinking and experimentation had begun in Britain &apos s oldest universities beginning in the seventeenth century at Oxford with the scientists Robert Hooke and Robert Boyle, and at Cambridge where Isaac Newton was Lucasian Professor of Mathematics and Physics. Moving into the 19th century, the objective of universities all across Europe evolved from teaching the &quot regurgitation of knowledge &quot to &quot encourag[ing] productive thinking. &quot In 1810, Humboldt convinced the King of Prussia to build a university in Berlin based on Friedrich Schleiermacher &apos s liberal ideas; the goal was to demonstrate the process of the discovery of knowledge and to teach students to &quot take account of fundamental laws of science in all their thinking.&quot
  </p>,
  <br />,
  <br />,
  <p>
    British universities of this period adopted some approaches familiar to the Italian and German universities, but as they already enjoyed substantial freedoms and autonomy the changes there had begun with the Age of Enlightenment, the same influences that inspired Humboldt. The Universities of Oxford and Cambridge emphasized the importance of research, arguably more authentically implementing Humboldt &apos s idea of a university than even German universities, which were subject to state authority. Overall, science (including mathematics) became the focus of universities in the 19th and 20th centuries.
  </p>,
]
`;
